content/articles/2018-07-talk-carpe-diem.markdown

```diff
@@ -0,0 +1,82 @@
+description: >
+  Seize the day and show off your work!
+categories:
+  - other
+photo:
+  url: https://unsplash.com/photos/iIWCjgK3704
+  author: Karine Germain
+---
+
+Summer conference season is in full swing, with [useR! 2018 in Brisbane](https://user2018.r-project.org) coming up next week!
+
+Conference talks are a great opportunity to help people learn about the cool and useful things you have built. A few concrete practices will make your marketing efforts more effective.
+
+*We assume your product meets your quality standards and that you have already documented it. This is about stuff you can do in ~1 hour as you prepare you talk that will have high payoff in terms of recruiting users. This is also not about how to give a great talk.*
+
+## Marketing is not a Four Letter Word
+
+First, let's deal with the word and concept of "marketing".
+
+> Marketing is the study and management of exchange relationships.
+
+https://en.wikipedia.org/wiki/Marketing
+
+If you have taken the trouble to build an R package, Shiny app, tutorial, or talk, presumably you would like people to know about it and use it! Marketing is the set of activities that help your target audience
+
+  * discover that your thing exists
+  * learn how your thing could benefit them
+
+Use your talk (and documentation and README, etc.) to accomplish these goals!
+
+Some creators take a peculiar pride in being "bad at marketing", but it is not some elusive skill. Nor is marketing ability anti-correlated with creating a high-quality product. You can acknowledge the value in marketing or you can declare that cultivating broader usage is not your goal. Pick one.
+
+## Where does your thing live?
+
+Determine the primary online home for your product. Possibilities:
+
+  * A custom website, possibly made with [pkgdown](http://pkgdown.r-lib.org), [blogdown](https://bookdown.org/yihui/blogdown/), [bookdown](https://bookdown.org/yihui/bookdown/), or, in the case of [Shiny](https://shiny.rstudio.com), a [running instance of the app itself](https://www.shinyapps.io)
+  * A development venue, such as a [GitHub repository](https://github.com)
+  * A [CRAN](https://cran.r-project.org/web/packages/index.html) package landing page
+
+Visit this home with the eyes of a newcomer and refine accordingly. If your thing has multiple online homes, make sure they link to each other, with a strong current towards the primary home.
+
+## Make a shortlink
+
+Unless your primary URL is already extremely short and memorable, consider creating a shortlink that is both. A link in a talk will be captured in cell phone photos and typed with fingers into phones. Make it easy on people!
+
+http://bit.ly
+
+## Title and closing slide
+
+Make a slide with the most important details on you and your thing. Include this very early in your talk (probably first?) and repeat some variant at the end. This slide should feature your shortlink and any relevant URLs or handles, e.g. Twitter, GitHub, website, or email address.
```

Comment:
This slide should feature:

* your name
* how to contact you (e.g. twitter handle, email)
* name of your thing
* how to find your thing (e.g. website, GitHub repo)

If you can squeeze in a quick description of your thing, or a visual summary of your talk, do so! That way people will get a little context for free when they come back to the photo.

Reply:
Not here, per se, but I think we might want to add something about visuals generally. Two examples that come to mind are @riannone's gt slides (though those aren't public, I guess), and Garrick Aden-Buie's ggplot2 deck (specifically the progression from 21-26 as it intermingles code and visuals nicely https://pkg.garrickadenbuie.com/trug-ggplot2/#21 — also I think they're reveal.js or xarigan, so it's not a matter of having fancy graphics programs or dealing with keynote).